modules/ROOT/pages/advanced/events.adoc

```diff
@@ -350,45 +358,49 @@ include::partial$misc/requires_5_1v.adoc[]
+| Fired before a notification is displayed. For information on the Notifications Specification (`NotificationSpec`), see: xref:advanced/creating-custom-notifications.adoc[Create custom notifications].
+include::partial$misc/requires_5_6v.adoc[]
 
 | OpenNotification
 | `{ notification: NotificationApi }`
-| Fired after a notification is displayed. For information on the Notifications API (`NotificationApi`), see: link:{baseurl}/advanced/creating-custom-notifications/[Create custom notifications]. include::partial$misc/requires_5_6v.adoc[]
+| Fired after a notification is displayed. For information on the Notifications API (`NotificationApi`), see: xref:advanced/creating-custom-notifications.adoc[Create custom notifications].
+include::partial$misc/requires_5_6v.adoc[]
 |===
 
 == Plugin events
 
 The following plugins provide events.
 
-* <<advancedtablesevents,Advanced Tables events>>
-* <<autosaveevents,Autosave events>>
-* <<charactermapevents,Character Map events>>
-* <<commentsevents,Comments events>>
-* <<exportevents,Export events>>
-* <<formatpainterevents,Format Painter events>>
-* <<fullscreenevents,Fullscreen events>>
-* <<importcssevents,Import CSS events>>
-* <<linkcheckerevents,Link checker events>>
-* <<listsevents,Lists events>>
-* <<mediaembedevents,Media Embed events>>
-* <<pasteevents,Paste events>>
-* <<permanentpenevents,Permanent Pen events>>
-* <<powerpasteevents,PowerPaste events>>
-* <<spellcheckerevents,Spell Checker events>>
-* <<spellcheckerproevents,Spell Checker Pro events>>
-* <<tableevents,Table events>>
-* <<visualblocksevents,Visual Blocks events>>
-* <<visualcharactersevents,Visual Characters events>>
-* <<wordcountevents,Word Count events>>
-
+* xref:advancedtablesevents[Advanced Tables events]
+* xref:autosaveevents[Autosave events]
+* xref:charactermapevents[Character Map events]
+* xref:commentsevents[Comments events]
+* xref:exportevents[Export events]
+* xref:formatpainterevents[Format Painter events]
+* xref:fullscreenevents[Fullscreen events]
+* xref:importcssevents[Import CSS events]
+* xref:linkcheckerevents[Link checker events]
+* xref:listsevents[Lists events]
+* xref:mediaembedevents[Media Embed events]
+* xref:pasteevents[Paste events]
+* xref:permanentpenevents[Permanent Pen events]
+* xref:powerpasteevents[PowerPaste events]
+* xref:spellcheckerevents[Spell Checker events]
+* xref:spellcheckerproevents[Spell Checker Pro events]
+* xref:tableevents[Table events]
+* xref:visualblocksevents[Visual Blocks events]
+* xref:visualcharactersevents[Visual Characters events]
+* xref:wordcountevents[Word Count events]
+
+[[advancedtablesevents]]
 === Advanced Tables events
 
 For events applicable to the Advanced Tables plugin, see: <<tableevents,Table events>>.
 
+[[autosaveevents]]
 === Autosave events
 
-The following events are provided by the link:{baseurl}/plugins/opensource/autosave/[Autosave plugin].
+The following events are provided by the xref:plugins/opensource/autosave.adoc[Autosave plugin].
```

Comment:
flatten the link to be xref:autosave.adoc

Reply:
See previous comment, TinyMCE 5 does not have a flat structure and cannot since we need to maintain compatibility with existing links.